packages/api/__tests__/index.js

```diff
@@ -1,5 +1,187 @@
-describe('Simple Jest Test', () => {
-  it('should run without crash', () => {
-    expect('Welcome').toEqual('Welcome');
+import mongoose from 'mongoose';
+import request from 'request-promise-native';
+
+import app from '../src';
+import { PORT } from '../config';
+import tokenService from '../src/services/token';
+import { User } from '../src/db';
+
+let savedUsers;
+const dummyUsers = [
+  {
+    firstName: 'Anirudh',
+    lastName: 'Nimmagadd',
+    email: 'anirudh.nimmagadda@gmail.com',
+    role: 'student',
+    batchId: mongoose.Types.ObjectId('111111111111111111111111'),
+  },
+  {
+    firstName: 'Bharani',
+    lastName: 'Dharani',
+    email: 'bharani.dharan@gmail.com',
+    role: 'student',
+    batchId: mongoose.Types.ObjectId('111111111111111111111112'),
+  },
+  {
+    firstName: 'Vipul',
+    lastName: 'Rawat',
+    email: 'vipul.rawat@gmail.com',
+    role: 'student',
+    batchId: mongoose.Types.ObjectId('111111111111111111111113'),
+  },
+];
+
+let server;
+const serverUrl = `http://localhost:${PORT}`;
+
+beforeAll((done) => {
+  jest.setTimeout(10000);
+  mongoose.connect(process.env.DB_URL, { useNewUrlParser: true }, () => {
```

Comment:
You can use your `config` files here too for importing constants.

Reply:
done.